BHoM/ComparisonConfig.cs

```diff
@@ -58,10 +58,11 @@ public class ComparisonConfig : IObject
 
         [Description("Number of fractional digits retained for individual property. If a property name matches a key in the dictionary, applies a rounding to the corresponding number of digits."
             + "\nSupports * wildcard in the property name matching. E.g. `{ { StartNode.Point.*, 2 } }`.")]
-        public virtual Dictionary<string, int> FractionalDigitsPerProperty { get; set; } = null; // e.g. { { StartNode.Point.X, 2 } } � can use * wildcard here.
+        public virtual Dictionary<string, int> FractionalDigitsPerProperty { get; set; } = null; // e.g. { { StartNode.Point.X, 2 } } � can use * wildcard here.
```

Comment:
```suggestion
        public virtual Dictionary<string, int> FractionalDigitsPerProperty { get; set; } = null; // e.g. { { StartNode.Point.X, 2 } } - can use * wildcard here.
```

@FraserGreenroyd - spotted this erroneously converted character

Reply:
Interesting, I wonder if a slightly different encoding was used on this object when it was set up. @alelom I'm guessing it's one of your objects because it's a ComparisonConfig, have you got any information you can share on your set up that might help me work out why BHoMBot didn't like it (and thus how to mitigate it happening next time)? :smile:

Reply:
When I write (text, rather than code) I tend to use the "en dash" symbol for hyphens: Alt + 0150 that returns `–` instead of the "minus" symbol `-`. I think this must've been caused by that. I'll be careful not to use it in code in the future.